src/platform/Linux/CHIPBluezHelper.cpp

```diff
@@ -997,7 +1001,9 @@ static void BluezHandleNewDevice(BluezDevice1 * device, BluezEndpoint * apEndpoi
         conn->mpDevice      = static_cast<BluezDevice1 *>(g_object_ref(device));
         conn->mpEndpoint    = apEndpoint;
         BluezConnectionInit(conn);
-
+        apEndpoint->mpPeerDevicePath = g_strdup(g_dbus_proxy_get_object_path(G_DBUS_PROXY(device)));
+        ChipLogDetail(DeviceLayer, "coonnected, insert, conn:%p, c1:%p, c2:%p and %s", conn, conn->mpC1, conn->mpC2,
```

Comment:
typo: connectected -> Connected

Can we avoid logging pointers?

Reply:
Just logging address and peer path now.